src/activity_actor.cpp

```diff
@@ -2384,7 +2384,7 @@ void insert_item_activity_actor::finish( player_activity &act, Character &who )
                 success = holster->put_in( it, item_pocket::pocket_type::CONTAINER,
                                            /*unseal_pockets=*/true ).success();
                 if( success ) {
-                    who.add_msg_if_player( string_format( _( "You insert your %1$s into the %2$s." ),
+                    who.add_msg_if_player( string_format( _( "You put your %1$s into the %2$s." ),
```

Comment:
```suggestion
                    //~ %1$s: item to put in the container, %2$s: container to put item in
                    who.add_msg_if_player( string_format( _( "You put your %1$s into the %2$s." ),
```

A comment like this will aid translators in knowing which string is which (since they may appear in a different order in some languages).